src/array_api_stubs/_2022_12/fft.py

```diff
@@ -18,13 +35,13 @@ def fft(
     Parameters
     ----------
     x: array
-        input array. Should have a floating-point data type.
+        input array. Should have a complex-valued floating-point data type.
     n: int
         length of the transformed axis of the output. If
 
-        - ``n`` is greater than the length of the input array, the input array is zero-padded to length ``n``.
-        - ``n`` is less than the length of the input array, the input array is trimmed to length ``n``.
-        - ``n`` is not provided, the length of the transformed axis of the output must equal the length of the input along the axis specified by ``axis``.
+        - ``n`` is greater than the length of the input array along ``axis``, the input array along ``axis`` must be zero-padded to length ``n``.
```

Comment:
In general, in order to refer to the number of elements along an axis, we have used, tmk, "size", rather than "length". We didn't adhere to that in the original FFT PR, likely as we more or less copied the phrasing from NumPy docs.

If we are comfortable adopting consistent language (i.e., using "size"), I can do a pass over this PR and update accordingly.

Reply:
Yup if you have time, thx 🙏

Reply:
Well I was trying to normalize things here, but in the other direction (using "length" instead of "size"). The reason I would prefer "length" to "size" is that `size` is an attribute on the array object that refers to the total number of elements. There was also a related discussion in NumPy https://github.com/numpy/numpy/pull/25140

Reply:
@asmeurer Thanks for that context. I am not sure I agree with the conclusions of that discussion, but I don't think this PR is the right place to change course. As "size" is the current term use throughout the specification when referring to the number of elements along a dimension, we should, IMO, continue to adhere to that convention in this PR.

As a follow-up, I suggest opening a separate issue outlining your concerns and arguments for changing the terminology. Should the reception prove positive, we can then make the changes across the entire specification more generally with potential inclusion in the v2024 revision.

Reply:
If this works for you, I can revert the s/size/length/ changes when I make some additional copy edits to align the language used in this PR with the rest of the specification.

Reply:
I support sticking to the current array API convention for now, and open a new issue for terminology discussion.

Reply:
Yes please proceed with pushing whatever changes are necessary to get this merged.